restapi/src/main/java/io/stargate/web/resources/RowResource.java

```diff
@@ -410,10 +416,8 @@ private static Object filterToValue(Object val, String column, Table tableData)
         .collect(Collectors.toList());
   }
 
-  private List<String> idFromPath(PathSegment id) {
-    MultivaluedMap<String, String> matrixParameters = id.getMatrixParameters();
-    List<String> values = new ArrayList<>(matrixParameters.keySet());
-    values.add(0, id.getPath());
-    return values;
+  private List<String> idFromPath(String path) {
+    String id = path.substring(path.lastIndexOf("/") + 1);
```

Comment:
I assume that it is not possible to get `/keyspaces/x/tables/y/1;2;3/` here?

Reply:
The path for this is `/keyspaces/ks1/tables/tbl1/rows/1;2;3`. For `/keyspaces/ks1/tables/tbl1` that would be picked up by `getAll()` and not go through here.

Reply:
What happens if one of the PK components is a string that contains the character `/`? Would it have to be URL-encoded, or is there a chance that it might cause a bug here?

Reply:
It should be URL encoded.